pandas/core/frame.py

```diff
@@ -78,8 +78,7 @@
                            OrderedDict, raise_with_traceback)
 from pandas import compat
 from pandas.compat.numpy import function as nv
-from pandas.util.decorators import (deprecate, Appender, Substitution,
-                                    deprecate_kwarg)
+from pandas.util.decorators import deprecate, Appender, Substitution
```

Comment:
It's deprecate that should be removed, not deprecate_kwarg 
(hence the travis failures)